proto/regen/ecocredit/v1alpha1/query.proto

```diff
@@ -58,6 +59,127 @@ service Query {
     option (google.api.http).get =
         "/regen/ecocredit/v1alpha1/params";
   }
+
+  // SellOrders queries a paginated list of all sell orders
+  rpc SellOrders(QuerySellOrdersRequest) returns (QuerySellOrdersResponse) {
+    option (google.api.http).get = "/regen/ecocredit/v1alpha1/marketplace/sell-orders";
```

Comment:
Can we use a single query for `SellOrders` and `SellOrder`, and make `sell-order-id` optional parameter?